src/main/java/com/stripe/model/Source.java

```diff
@@ -121,10 +121,10 @@
   Owner owner;
 
   @SerializedName("p24")
-  SourceTypeP24 p24;
+  P24 p24;
 
   @SerializedName("paper_check")
-  SourceTypePaperCheck paperCheck;
+  PaperCheck paperCheck;
```

Comment:
This is not public

Reply:
I also find `PaperCheck` and `SepaCreditTransfer` attached to `Customer` object returned from `Invoice` too..

Reply:
Sure but things are secret and gated and we should avoid leaking them in public libraries really that's all

Reply:
Ack. Was just concerned if we actually hide this, we will drop the entire hash object from `source`. Previously we had this Map<String, Object> with `sourceTypeData`, so people could still access them. But if it' entirely internal usage, then yup I'll try to undocument them.

Reply:
It's not really internal usage, it's in private beta for some merchants. But this is sadly how beta work with static languages, you can't use the library for those.